core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -106,13 +105,13 @@ public void start() {
             location = "Infinispan-SingleFileStore";
 
          file = new File(location, ctx.getCache().getName() + ".dat");
-         if (!file.exists()) {
+         if (!SecurityActions.fileExists(file)) {
             File dir = file.getParentFile();
             if (!dir.mkdirs() && !dir.exists()) {
```

Comment:
mkdirs() and exists() also require PrivilegedActions.

Reply:
Sorry @pferraro, I had fixed that, but I'm trying to avoid branch switch issues with IntelliJ by working on 9.4.x from my downstream working directory, and I keep pushing to the wrong remote :)